articles/billing/billing-enterprise-api.md

```diff
@@ -40,6 +40,9 @@ A Swagger endpoint is available [here](https://consumption.azure.com/swagger/ui/
 
 * **Price Sheet** - The [Price Sheet API](https://docs.microsoft.com/rest/api/billing/enterprise/billing-enterprise-api-pricesheet) provides the applicable rate for each Meter for the given Enrollment and Billing Period. 
 
+## Data Freshness:
+Etags will be returned in the response of all the above API. A change in Etag indicates the date has been refreshed.  In subsequent calls to the same API with the same parameters, pass the captured Etag with the key “If-None-Match” in the header of http request. The response status code would be "NotModified" if the data has not been refreshed any further. API will not return any data in the case there is no change.
```

Comment:
Lets mention the other case where data has changed. In that case API will return full data (not delta)